Tests/MMELocationManagerTests.m

```diff
@@ -42,31 +74,6 @@ - (void)testLocationManagerStartsMonitoringRegions {
     XCTAssert(self.locationManager.monitoredRegions > 0);
 }
 
-- (void)testRegionDistanceFilter {
```

Comment:
Invalid Test] testRegionDistanceFilter

Problem:
testRegionDistanceFilter is looking to validate that in the instance of lots of micro location updates, that we’re not overly updating our regions.

Problem:
This test is making comparisons of state synchronously, while the action is asynchronous meaning the comparisons are invalid.

Why:
https://github.com/mapbox/mapbox-events-ios/blob/master/MapboxMobileEvents/MMELocationManager.m#L215

Docs:
 *      This is done asynchronously and may not be immediately reflected in monitoredRegions.
 */
- (void)startMonitoringForRegion:(CLRegion *)region API_AVAILABLE(ios(5.0), macos(10.8)) API_UNAVAILABLE(watchos, tvos);

Approach:
Attempted to inject a spy LocationManager to detect when this was updated, unfortunately this was indeterminant due to LocationManager caching last location. Making this dependent on any other location manager which may have run prior.

Change:
- Remove Test

Alternate Options:
Fully extract out Locationmanager behind a protocol and pass that interface. This would require some implementation behaviors which may not fully match CLLocationManager.